microsoft-edge/devtools-guide-chromium/evaluate-performance/reference.md

```diff
@@ -130,117 +146,133 @@ To throttle the network while recording:
 
 
 <!-- ------------------------------ -->
 #### Search activities
 
-Press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS) to open the search box at the bottom of the **Performance** tool.
+To open the search box at the bottom of the **Performance** tool, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS):
+
+![The search box](./reference-images/search-regex.png)
 
-![The search box](./reference-images/evaluate-performance-performance-search-regex.png)
 
 To navigate activities that match your query:
 
 1. Click the **Previous** (![Previous](./reference-images/previous-icon.png)) and **Next** (![Next](./reference-images/next-icon.png)) buttons.
 
 1. To select the next activity, press **Enter**.  To select the previous activity, press **Shift+Enter**.
 
-To modify query settings:
+
+To modify the query settings:
 
 *  To make the query case-sensitive, click the **Case sensitive** (![Case sensitive](./reference-images/search-case-icon.png)) button.
 
 *  To use a regular expression in your query, click the **Regex** (![Regex](./reference-images/search-regex-icon.png)) button.
 
+
 To hide the search box, click **Cancel**.
 
 
 <!-- ------------------------------ -->
 #### View main thread activity
 
-Use the **Main** section to view activity that occurred on the main thread of the page.
+Use the **Main** section to view the activity that occurred on the main thread of the page:
 
-![The Main section](./reference-images/evaluate-performance-performance-main-zoomed.png)
+![The Main section](./reference-images/main-zoomed.png)
 
-Select an event to view more information about it in the **Summary** tab.  DevTools outlines the selected event.
+Select an event to view more information about it in the **Summary** tab.  DevTools outlines the selected event:
 
-![More information about the anonymous function in the Summary tab](./reference-images/evaluate-performance-performance-summary-me.png)
+![More information about the anonymous function in the Summary tab](./reference-images/summary-me.png)
 
 DevTools represents main thread activity with a flame chart.  The x-axis represents the recording over time.  The y-axis represents the call stack.  The events on top cause the events below it.
 
-![A flame chart](./reference-images/evaluate-performance-performance-main-flame-chart.png)
+![A flame chart](./reference-images/main-flame-chart.png)
 
-In the previous figure, a `click` event caused a `Function Call` in `activitytabs.js` on line 53.  Below `Function Call`, review that an anonymous function was run.  The anonymous function requested `a`, which requested `wait`, which requested `Minor GC`.
+In the previous figure, an `input` event caused a `Function Call`, which ran an anonymous function that called `filterByCamera` and then `populateGallery`.  The `populateGallery` function then made DOM changes by calling `innerHTML`.
```

Comment:
```suggestion
In the previous figure, an `input` event caused a `Function Call`, which ran an anonymous function that called `filterByCamera` and then `populateGallery`.  The `populateGallery` function then made DOM changes by calling `setInnerHTML`.
```
@captainbrosset fyi (i'm editing in VS Code)
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference?branch=pr-en-us-2945#view-main-thread-activity